eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -116,7 +116,7 @@ jobs:
           version: "$(DotNetCoreSDKVersion)"
       - script: >-
           dotnet test eng/service.proj --filter TestCategory!=Live --framework $(TestTargetFramework) --logger "trx;LogFileName=$(TestTargetFramework).trx" 
-          /p:ServiceDirectory=${{ parameters.ServiceDirectory }} /p:IncludeSrc=false /p:Configuration=$(BuildConfiguration)
+          /p:ServiceDirectory=${{ parameters.ServiceDirectory }} /p:IncludeSrc=false /p:IncludeSamples=false /p:Configuration=$(BuildConfiguration)
```

Comment:
Is there a reason you need to exclude the samples as part of this work?

Reply:
We don't need the samples in the docs.

Reply:
This is only the test run the docs shouldn't be getting generated from test command line.

Reply:
I had to add this due to changes I made in the service.proj. I added a property for Including/excluding samples so I can exclude samples in the docs. I adding that property here to ensure samples are not included or else they will break CI on the NetFramework test